src/main/java/project/service/CarServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package project.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import project.dao.CarDao;
+import project.lib.Inject;
+import project.lib.Service;
+import project.models.Car;
+import project.models.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        List<Driver> drivers = new ArrayList<>();
+        drivers.add(driver);
+        car.setDrivers(drivers);
+        carDao.update(car);
```

Comment:
it seems you are rewriting the data here, respectively, only one driver can be attached to each car